feature-pack/src/main/resources/modules/system/layers/fuse/org/apache/camel/component/braintree/main/module.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<module xmlns="urn:jboss:module:1.1" name="org.apache.camel.component.braintree">
+  <resources>
+    <artifact name="${org.apache.camel:camel-braintree}" />
+  </resources>
+  <dependencies>
+    <module name="javax.api" />
+    <module name="org.slf4j" />
+    <module name="com.braintreepayments.gateway.braintree" />
```

Comment:
This can probably include instead of reference the com.braintreepayments.gateway.braintree module. We usually do this, when there is no need to share com.braintreepayments.gateway.braintree